### PR TITLE
updated refed page

### LINKED ISSUE #803: Update /refed
The refed page should mention the flipper and mod including what features it has and that there is an how to get premium. Also Test-Premium

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #803

### CODE REVIEW

**Comment by matthias-luger** [comment]
@Ekwav Is this enough? Feel free to add/change things to the text.

File: pages/refed.tsx [hunk 1]
```diff
@@ -17,13 +19,38 @@ function Refed() {
...
+                            <Tooltip
+                                content={
+                                    <span>
+                                        <HelpIcon />
+                                    </span>
+                                }
+                                type="hover"
+                                tooltipContent={
+                                    <p>
+                                        To connect your Minecraft account, search and click yourself in the search bar. Afterwards click "Your? Claim account."
```
Comment (Ekwav) [inline pages/refed.tsx]:
```suggestion
                                        To connect your Minecraft account, search and click yourself in the search bar. Afterwards click "You? Claim account."
```
thats not the text thats on the player page

File: pages/refed.tsx [hunk 2]
```diff
@@ -17,13 +19,38 @@ function Refed() {
...
+                                        to get a full explanation.{' '}
+                                    </p>
+                                }
+                            />{' '}
+                            to get 1 hour of <Link href="/premium">premium</Link> for free to test our service including our{' '}
+                            <Link href="/flipper">auction flipper</Link>.
```
Comment (Ekwav) [inline pages/refed.tsx]:
```suggestion
                            to get 24 hours of our <Link href="/premium">premium plan</Link> for free. That includes our{' '}
                            <Link href="/flipper">advanced auction flipper</Link>.
```

File: pages/refed.tsx [hunk 3]
```diff
@@ -17,13 +19,38 @@ function Refed() {
...
+                        </p>
+                        <p>
+                            We also provide a mod to use the flipper ingame. You can download it in the <b>#mod-releases</b> channel on our{' '}
```
Comment (Ekwav) [inline pages/refed.tsx]:
```suggestion
                            We also provide a Minecraft mod to use the flipper in game. You can download it in the <b>#mod-releases</b> channel on our{' '}
```

File: pages/refed.tsx [hunk 4]
```diff
@@ -17,13 +19,38 @@ function Refed() {
...
+                            <a href="https://discord.gg/wvKXfTgCfb">
+                                <span style={{ color: '#7289da' }}>Discord</span>
+                            </a>
+                            . For help check out the <b>#faq</b> channel or ask in <b>#support</b>.
```
Comment (Ekwav) [inline pages/refed.tsx]: the faq is also on /flipper it might be better to link there since some people don't use discord

File: pages/refed.tsx [hunk 5]
```diff
@@ -17,13 +19,38 @@ function Refed() {
...
+                        </p>
                         <hr />
                         <p>Login with Google:</p>
                         <GoogleSignIn onAfterLogin={() => {}} />
                         <p>
-                            We use Google accounts because they are more secure than requiring a separate login. We use your Google Id and email in compliance
-                            with our <a href="https://coflnet.com/privacy">privacy policy</a> (i.e. to know what settings you made and contact you in case we
-                            need to)
+                            We use Google accounts because they are more secure than requiring a separate login. We use your Google Id and email. (i.e. to know
```
Comment (Ekwav) [inline pages/refed.tsx]: why was this changed, the new version doesn't make sense. Nobody uses the Google Id or email of our users, we at most store them

**Review by Ekwav — CHANGES_REQUESTED** [review]

File: pages/refed.tsx (hunk 5 above)
Comment (matthias-luger) [reply pages/refed.tsx]: It is the same text as before, I just removed the part with the privacy policy as it's already mentioned and linked right obove it because of the google login.
image

Comment (matthias-luger) [reply pages/refed.tsx]: Should the part with the google id and email be removed entirely?

Comment (Ekwav) [reply pages/refed.tsx]:
```suggestion
                            Settings you make are tied to your google account to sync across devices and into the Minecraft mod. 
```
something like this?
couldn't add the suggestion for more lines since I didn't originally mark them.

**Review by Ekwav — APPROVED** [review]
